.github/workflows/build-and-test.yml

```diff
@@ -42,14 +42,19 @@ jobs:
             fail-fast: false
             max-parallel: 4
             matrix:
-                python-version: [3.7, 3.8]
+                python-version: [3.7, 3.8, 3.9]
                 python-architecture: [x64]
                 os: [windows-latest, macos-latest]
                 include:
                     - python-version: 3.7
                       numpy: "numpy=1.15"  # fuzzy assertion in conda is single '='
+                      gdal: "gdal==3.1.2"
                     - python-version: 3.8
                       numpy: "numpy=1.16"
+                      gdal: "gdal==3.2.0"
+                    - python-version: 3.9
+                      numpy: "numpy=1.20"
+                      gdal: "gdal==3.3.0"
```

Comment:
Should this be 3.3.1 given the bug fix of the environment variable issue?

Reply:
yes thanks!